### PR TITLE
added touch icons for apple, android and windows live tiles

### PR DESCRIPTION
**Pull request by apinner**

### CODE REVIEW

File: src/app/services/theme-manager.provider.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ class ThemeManagerProvider {
       $rootScope.theme = this.theme;
       $rootScope.assetsFolder = 'app/themes/' + this.theme + '/assets';
       $rootScope.favicon = '/app/themes/' + this.theme + '/assets/favicon.png';
+      $rootScope.touchicons = '/app/themes/' + this.theme + '/assets/icons';
```
Comment (kurtommy) [inline src/app/services/theme-manager.provider.js]: add this var

$rootScope.themeFolder = '/app/themes/' + this.theme;

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,26 @@
...
+    <link rel="apple-touch-icon" sizes="120x120" href="{{touchicons}}/apple-touch-icon-120x120.png">
+    <link rel="apple-touch-icon" sizes="144x144" href="{{touchicons}}/apple-touch-icon-144x144.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="{{touchicons}}/apple-touch-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="{{touchicons}}/apple-touch-icon-180x180.png">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-194x194.png" sizes="194x194">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-96x96.png" sizes="96x96">
+    <link rel="icon" type="image/png" href="{{touchicons}}/android-chrome-192x192.png" sizes="192x192">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="{{touchicons}}/manifest.json">
```
Comment (kurtommy) [inline src/index.html]: use this path
<link rel="manifest" href="{{themeFolder}}/manifest.json">

manifest.json il related to the whole theme and not to icons. Move the manifest in the theme folder at first level please.

File: src/index.html [hunk 3]
```diff
@@ -9,6 +9,26 @@
     <meta name="apple-mobile-web-app-capable" content="yes"/>
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <link rel="shortcut icon" type="image/png" ng-href="{{favicon}}"/>
+    <link rel="apple-touch-icon" sizes="57x57" href="{{touchicons}}/apple-touch-icon-57x57.png">
```
Comment (kurtommy) [inline src/index.html]: rename every instance of touchicons in toucIcons

File: src/index.html (hunk 2 above)
Comment (apinner) [inline src/index.html]: @kurtommy I've made all of the other changes, but I don't follow this. Use which path?

manifest.json just defines icons for android chrome, why does it need to be at the first level?

Comment (apinner) [inline src/index.html]: Can you clarify @kurtommy? Still not sure what you mean.

Comment (kurtommy) [inline src/index.html]: @apinner there are a lot of keys not related to icons, and lot more are coming soon. Eg
`"start_url": "/index.html",
  "display": "standalone",
  "orientation": "landscape"`

Comment (apinner) [inline src/index.html]: @kurtommy i'll add a zip of the icons later and you can add them how you like if that is easier?

**Pull request closed without merging**